Gaijinizer patch/Map452.txt

```diff
@@ -4,696 +4,858 @@
 
 > BEGIN STRING
 ……………声がいきなり裏返った……
-> CONTEXT: Map452/events/1/pages/0/66/Dialogue/kazuya < UNTRANSLATED
-
+> CONTEXT: Map452/events/1/pages/0/66/Dialogue/kazuya
+Her voice just squeaked...#Ron
 > END STRING
 
 > BEGIN STRING
 どうも、彼女にとってはイヤな話題らしい……
 
 あまり刺激しないように、これ以上の追求はやめておこう…
-> CONTEXT: Map452/events/1/pages/0/68/Dialogue < UNTRANSLATED
+> CONTEXT: Map452/events/1/pages/0/68/Dialogue
+It seems this is a touchy subject for her.
 
+I'll avoid getting her too worked up and just leave it at
+that.#Ron
 > END STRING
 
 > BEGIN STRING
 さっき『双子』がどうこうって言ってたが……
 
 するとお前も、アバドーンに頼まれて襲いに来た
 別世界からの刺客か…！？
-> CONTEXT: Map452/events/1/pages/0/72/Dialogue/other1 < UNTRANSLATED
-
+> CONTEXT: Map452/events/1/pages/0/72/Dialogue/other1
+You said something before about "twins". Are you
+assassins from another world tasked by Abaddon to
+come attack us?!#Ron
 > END STRING
 
 > BEGIN STRING
 その通り……
 
 あんな得体の知れないヤツを信用したわけじゃないけど
 人間を襲えるっていうなら、話は別よ…♡
-> CONTEXT: Map452/events/1/pages/0/77/Dialogue/enemy38 < UNTRANSLATED
-
+> CONTEXT: Map452/events/1/pages/0/77/Dialogue/enemy38
+Exactly.
+It's not like I trusted those weirdos, but it's
+a different story if we're talking about
+attacking humans.#RonIffyWording
 > END STRING
 
 > BEGIN STRING
 この世の人間は、全部我々魔族のオモチャ…♡
 
 アンタも例外じゃないわよ、\\N[1]
 この私に見つかった以上ね…！
-> CONTEXT: Map452/events/1/pages/0/82/Dialogue/enemy37 < UNTRANSLATED
-
+> CONTEXT: Map452/events/1/pages/0/82/Dialogue/enemy37
+All the humans of this world are the playthings
+of demonkind, and you, \\N[1], are no
+exception. Not now that I've found you! ♡#Ron
 > END STRING
 
 > BEGIN STRING
 まずはアンタの力、試させてもらうわ！
 
 行きなさい！\\C[2]ブラックドッグ！！
 その人間と、たっぷり遊んでやるのよ！
-> CONTEXT: Map452/events/1/pages/0/87/Dialogue/enemy37 < UNTRANSLATED
+> CONTEXT: Map452/events/1/pages/0/87/Dialogue/enemy37
+But first I'll test your strength!
 
+Come, \\C[2]Black Dog!!
+It's time to play with that human!#RonLocalization
 > END STRING
 
 > BEGIN STRING
 "EX4　ブラックドッグ戦"
-> CONTEXT: Map452/events/1/pages/0/94/InlineScript/1:24 < UNTRANSLATED
-
+> CONTEXT: Map452/events/1/pages/0/94/InlineScript/1:24
+"EX4　Black Dog Battle"#Ron
 > END STRING
 
 > BEGIN STRING
 ハッ……ハッ……
-> CONTEXT: Map452/events/1/pages/0/118/Dialogue/enemy37 < UNTRANSLATED
-> CONTEXT: Map465/events/29/pages/1/0/Dialogue/enemy37 < UNTRANSLATED
-> CONTEXT: Map465/events/29/pages/2/0/Dialogue/enemy37 < UNTRANSLATED
-
+> CONTEXT: Map452/events/1/pages/0/118/Dialogue/enemy37
+> CONTEXT: Map465/events/29/pages/1/0/Dialogue/enemy37
+> CONTEXT: Map465/events/29/pages/2/0/Dialogue/enemy37
+*pant*... *pant*...#Ron
 > END STRING
 
 > BEGIN STRING
 ヴァソパイアのかけ声とともに現れたのは
 巨大な黒のシルエットをした犬のような悪魔……
 
 その髪のような毛先は、先端が黄金色に輝いている……
-> CONTEXT: Map452/events/1/pages/0/120/Dialogue < UNTRANSLATED
-
+> CONTEXT: Map452/events/1/pages/0/120/Dialogue
+What appeared at the same time as the vasopire's shout
+was a dog-like demon casting a large black silhouette,
+the tips of what appeared to be its hair glittering with
+a golden hue.#Ron
 > END STRING
```

Comment:
`vasopire`? Think you meant 'vanpire' here for consistency unless I'm missing something.